src/source/Main.scala

```diff
@@ -282,6 +285,15 @@ object Main {
     } else {
       None
     }
+    val objcSwiftBridgingHeaderWriter = if (objcSwiftBridgingHeaderOut.isDefined) {
```

Comment:
Is there a reason to treat this as an independent generator rather than a part of ObjC generation?  From a code structure point of view I'm glad to have `SwiftBridgingHeaderGenerator.scala` be separate, but from a user point of view does it ever make sense to generate the header other than at the same time as generating ObjC?  If the two were combined, with the bridging header being an optional output, then it would also make sense to have a single output directory, and not have to deal with the cases where the two output directories differ.